app/uk/gov/hmrc/cipphonenumberverification/connectors/GovUkConnector.scala

```diff
@@ -43,15 +42,15 @@ class GovUkConnector @Inject()(httpClient: HttpClientV2, config: AppConfig)
       .execute[Either[UpstreamErrorResponse, HttpResponse]]
   }
 
-  def sendPasscode(passcode: PhoneNumberAndOtp)(implicit hc: HeaderCarrier): Future[Either[UpstreamErrorResponse, HttpResponse]] = {
+  def sendPasscode(phoneNumber: String, passcode: String)(implicit hc: HeaderCarrier): Future[Either[UpstreamErrorResponse, HttpResponse]] = {
```

Comment:
Why did we change this from PhoneNumberAndOtp to separate parameters

Reply:
I was having issues with the unit testing of the date parameter, however its not an issue anymore, therefore I have put back the data object as the parameter.

Reply:
I have put back the PhoneNumberPasscodeData parameter (the new data object).  as this ticket is getting merged before CAV-330.